src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -51,13 +53,27 @@ public FeedbackResponseRow(int fbIndex, int personIndex, String personType,
                 String showCommentTo = StringHelper.removeEnclosingSquareBrackets(frc.showCommentTo.toString());
                 String showGiverNameToString = StringHelper.removeEnclosingSquareBrackets(frc.showGiverNameTo.toString());
                 String recipientName = results.getNameForEmail(response.recipient);
-                String giverEmail = frc.giverEmail;
-                Map<String, String> instructorEmailNameTable = results.instructorEmailNameTable;
-                FeedbackResponseCommentRow responseRow = new FeedbackResponseCommentRow(frc,
+                String giverEmail = frc.commentGiver;
+                Map<String, String> commentGiverEmailToNameTable = results.commentGiverEmailToNameTable;
+                FeedbackResponseCommentRow responseCommentRow = new FeedbackResponseCommentRow(frc,
                         giverEmail, giverName, recipientName, showCommentTo, showGiverNameToString, responseVisibilities,
-                        instructorEmailNameTable, results.getTimeZone(), question);
-                responseRow.enableEditDelete();
-                this.responseComments.add(responseRow);
+                        commentGiverEmailToNameTable, results.getTimeZone(), question);
+                boolean isInstructorGiver = results.roster.isInstructorOfCourse(giverEmail);
+                // Instructor cannot edit/delete feedback participant's comments from results page
```

Comment:
Is this necessary?

Let make thing simple. By right, if it is from feedback participant (frcs.isCommentFromFeedbackParticipant), then is it not editable. Instructor can always masquerade as the student to moderate the comment.

Reply:
The UI related class should not contain any logic like permission checking.

Reply:
Instructor can edit comment given by another instructor?

Reply:
> Instructor can edit comment given by another instructor?

That is out of the scope of this PR?

In the result page in instructor side, they should not edit feedback's participant (either student, instructor or team)'s comment.